src/components/admin/user-accounts-tab.yml

```diff
@@ -90,3 +90,48 @@ userAccountsTab:
     de: |
       Wähle den Speicherplan aus, den du zuweisen möchtest. Keine Auswahl bedeuted
       Löschung {selectedItemCount, plural, =1 {eines im Benutzerkonto vorhandenen Speicherplans} other {von in den Benutzerkonten vorhandenen Speicherplänen}}.
+  roleInfoMarkdown:
+    en: |
+      # User
+
+      Users can create and edit public documents, as long as they are not protected or archived as well as upload public files.
+
+      # Accredited Author
+
+      Accredited authors can create protected public documents with exclusive editing permissions.
+
+      # Maintainer
+
+      Maintainers have extended content management permissions, such as reviewing, approving, protecting or archiving public documents.
```

Comment:
"Maintainers have extended content management permissions over public content and can curate documents and the media library resources. For documents these permissions include reviewing, approving, protecting or archiving, as well as hard-deleting sections." would be maybe an alternative?

Reply:
Same here.